src/lib/data/joinstr.json

```diff
@@ -0,0 +1,71 @@
+{
+  "title": "Coinjoins.org - Joinstr Review",
+  "wallet_name": "joinstr",
+  "protocol_name": "joinstr",
+  "wallet_logo": "joinstr",
+  "tags": [
+    {
+      "title": "Fourth Gen",
+      "type": "gen",
+      "description": "A fourth generation coinjoin protocol is a term used to describe protocols that have decentralized the coordinator model leveraging Nostr relays"
+    },
+    {
+      "title": "Alpha",
+      "type": "status",
+      "description": "Joinstr is alpha software and it's recommended to only use it in test networks."
+    }
+  ],
+  "hero": "Joinstr is a coinjoin protocol with a decentralized coordinator model on Nostr relays. Participants can use multiple relays in the same coinjoin round.",
+  "github": "https://gitlab.com/1440000bytes/joinstr/",
+  "telegram": "https://t.me/joinstr",
+  "website": "https://joinstr.xyz/",
+  "basics": [
+    {
+      "id": "release_date",
+      "title": "Release Date",
+      "body": "Proof of Concept: August 20, 2022"
+    },
+    {
+      "id": "platform",
+      "title": "Platform",
+      "body": "Web, CLI and Electrum Desktop Plugin"
+    },
+    {
+      "id": "programming_language",
+      "title": "Code",
+      "body": "Python"
+    },
+    {
+      "id": "address_type",
+      "title": "Address Type",
+      "body": "Segwit Native (bc1)"
+    }
+  ],
+  "overview": {
+    "point_1": "Joinstr aims to decentralize the coinjoin coordinator to solve the censorship resistance problem of other centralized protocols.",
+    "point_2": "It is currently not available on the Bitcoin mainnet network and using it with real money is strongly discouraged.",
+    "point_3": "Users start by selecting an input to be part of the next coinjoin round, with the process commencing once the required number of participants is reached.",
+    "point_4": "The Nostr relay coordinator receives an output of the same denomination from each participant with the sighash flag ALL|ANYONECANPAY, followed by each participant registering and signing the previously selected input.",
+    "point_5": "Each event is published with a new Nostr public key and a new IP address using the RISEUP API free service (not yet implemented). Tor is optional.",
```

Comment:
Riseup API is implemented in electrum plugin

Reply:
added